core/shared/src/main/scala/matryoshka/patterns/ListF.scala

```diff
@@ -50,7 +50,7 @@ object ListF {
   def seqIso[A] = AlgebraIso[ListF[A, ?], Seq[A]] {
     case ConsF(h, t) => h +: t
     case NilF()      => Seq.empty
-  } (s => s.headOption.fold[ListF[A, Seq[A]]](NilF())(ConsF(_, s.tail)))
+  } (s => s.headOption.fold[ListF[A, Seq[A]]](NilF())(ConsF(_, s.drop(1))))
```

Comment:
Yeah, that’s a good point. I was looking for an `uncons: Option[(A, List[A])]` and was surprised I  couldn’t. Pattern-match it is.